src/node/ext/ldap/filter.rst

```diff
@@ -90,25 +90,36 @@ objects to LDAP filters.::
 
     >>> criteria = dict(sn=u'meier\xe4', cn='sepp')
     >>> filter = LDAPDictFilter(criteria, or_search=True)
-    >>> filter
-    LDAPDictFilter(criteria={'cn': 'sepp', 'sn': u'meier\xe4'})
+    >>> filter.criteria['sn'] == u'meier\xe4'
+    True
+    >>> filter.criteria['cn'] == 'sepp'
+    True
 
-    >>> str(filter)
-    '(|(cn=sepp)(sn=meier\xc3\xa4))'
+    >>> str(filter) in ['(|(sn=meierä)(cn=sepp))', '(|(cn=sepp)(sn=meier\xc3\xa4))']
```

Comment:
I can not follow here? why check both 'ä' variants?

Reply:
The representation differs in Python 2 vs. 3. We don't care what exactly it is, so we allow both.